cmd/apprepository-controller/server/controller.go

```diff
@@ -134,7 +134,7 @@ func NewController(
 
 	log.Info("Setting up event handlers")
 	// Set up an event handler for when AppRepository resources change
-	apprepoInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+	_, err = apprepoInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
Seems odd that we're not using the result (but then, weren't before the change either).

Reply:
Yep, I don't know why we weren't using it... and why the golangci-lint linter hadn't complained until this PR :S